packages/components/filters/src/filters.tsx

```diff
@@ -1,22 +1,223 @@
    */
-  label?: string;
+  appliedFilters: TAppliedFilter[];
+  /**
+   * configuration for the available filters.
+   */
+  filters: TFilterConfiguration[];
+  /**
+   * optional configuration for filter groups.
+   */
+  filterGroups?: TFilterGroupConfiguration[];
+  /**
+   * controls the `clear all` (added filters) button from the menu list
+   */
+  onClearAllRequest: () => void;
+  /**
+   * optional callback trigger to add a new filter.
+   */
+  onAddFilterRequest?: Function;
+  /**
+   * function to render a search input, selectable from applicable UI Kit components.
+   */
+  renderSearchComponent: () => ReactNode;
+};
+
+const horizontalDividerStyles = css`
+  width: 100%;
+  height: 1px;
+  border: 1px solid ${designTokens.colorNeutral90};
+  margin-top: ${designTokens.spacing25};
+  margin-bottom: ${designTokens.spacing30};
+`;
+
+const verticalDividerStyles = css`
+  width: 1px;
+  height: ${designTokens.spacing30};
+  background-color: ${designTokens.colorNeutral90};
+  margin: 0 ${designTokens.spacing30} 0 ${designTokens.spacing30}!important;
+`;
+
+const FiltersButton = ({ onAddFilters }: { onAddFilters: () => void }) => {
+  const intl = useIntl();
+  return (
+    <FlatButton
+      data-testid="filters-button"
+      label={intl.formatMessage(messages.filtersButtonLabel)}
+      icon={<FilterIcon />}
+      onClick={onAddFilters}
+    />
+  );
+};
+
+const AddFilterButton = () => {
+  const intl = useIntl();
+  return (
+    <FlatButton
+      data-testid="add-filter-button"
+      label={intl.formatMessage(messages.addFiltersButtonLabel)}
+      icon={<PlusBoldIcon />}
+    />
+  );
+};
+
+//Removes filters from being visible & clears all data
+const ClearAllFiltersButton = ({
+  onClearAllRequest,
+}: {
+  onClearAllRequest: () => void;
+}) => {
+  const intl = useIntl();
+  return (
+    <>
+      <div css={verticalDividerStyles}> </div>
+      <FlatButton
+        icon={<CloseBoldIcon />}
+        label={intl.formatMessage(messages.clearAllFiltersButtonLabel)}
+        onClick={onClearAllRequest}
+        tone="secondary"
+      />
+    </>
+  );
 };
 
 function Filters(props: TFiltersProps) {
+  const [showAddFilters, setShowAddFilters] = useState(false);
+
+  const handleFiltersClick = () => {
+    setShowAddFilters((currState) => !currState);
+  };
+
   return (
-    <div
-      style={{
-        display: 'flex',
-        gap: designTokens.spacing40,
-      }}
-    >
-      {props.label && <span>{props.label}</span>}
-    </div>
+    <>
+      <Spacings.Inline scale="m" alignItems="center">
+        {props.renderSearchComponent()}
+        <FiltersButton onAddFilters={handleFiltersClick} />
+      </Spacings.Inline>
+      <div css={horizontalDividerStyles} />
+      {showAddFilters && (
+        <Spacings.Inline scale="s" alignItems="center">
+          <Popover.Root>
+            <Popover.Trigger asChild>
+              <div>
+                <AddFilterButton />
+              </div>
```

Comment:
oh, I didnt realize that a wrapping div is all that's necessary to make the trigger work as expected, interesting

Reply:
on closer inspection, it looks like the `div` is taking up more height than the button, which is causing this button to be out of alignment with the "clear all" button.
```suggestion
              <div css={{display: 'inline-flex'}}>
                <AddFilterButton />
              </div>
```
This should make it so that the buttons align.